alvr/server/cpp/platform/linux/EncodePipeline.cpp

```diff
@@ -14,9 +14,9 @@ extern "C" {
 
 void alvr::EncodePipeline::SetParams(FfiDynamicEncoderParams params) {
   if (params.updated) {
-    encoder_ctx->bit_rate = params.bitrate_bps;
-    encoder_ctx->framerate = AVRational{(int)params.framerate, 1};
-    encoder_ctx->rc_buffer_size = params.bitrate_bps / params.framerate * 1.1;
+    encoder_ctx->bit_rate = params.bitrate_bps / params.framerate * 60;
+    encoder_ctx->framerate = AVRational{60, 1};
+    encoder_ctx->rc_buffer_size = encoder_ctx->bit_rate / 60 * 1.1;
```

Comment:
change to 
```encoder_ctx->rc_buffer_size = encoder_ctx->bit_rate / (60 * 1.1);```
to resolve lint about lost of precision or something else

Reply:
Can you post the lint output? My compiler isn't complaining here.

Reply:
````clang-tidy: Result of integer division used in a floating point context; possible loss of precision```